### PR TITLE
Add Flatpak aarch64 builds

### PR DESCRIPTION
**Pull request by tytan652**
### Description

Depends on:
- #8026 

#### Add aarch64 to the manifest:

Disable SVT-AV1 on non-x86_64 and add CEF arm64, both setup with shell commands to avoid duplicating modules.

#### Add aarch64 to CIs:

Building through QEMU emulation is really slow. So to avoid timing out
the job (6h), the original job is split in two.

The first job build dependencies (4-5h), if the cache already exist the
build is skipped.

And the second job build CEF and OBS Studio by relying on the cache of the
first job (2-3h).

This separation is done by modifying the manifest in the first job to
build only dependencies with a fake "obs" module/command.

aarch64 builds are not enabled by default on the main workflow, it can
be enabled with a dedicated label on PRs.

### Motivation and Context

Publish Flatpak aarch64 builds.

### How Has This Been Tested?

CI with a PR on my fork: https://github.com/tytan652/obs-studio/pull/8 (aarch64 artifacts availlable)

### Types of changes

- CI

### Checklist:

- [x] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [x] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [x] My code is not on the master branch.
- [x] The code has been tested.
- [x] All commit messages are properly formatted and commits squashed where appropriate.
- [x] I have included updates to all appropriate documentation.

### CODE REVIEW

File: .github/workflows/flatpak.yml [hunk 1]
```diff
@@ -91,7 +163,7 @@ jobs:
 
       - name: Commit screenshots to the OSTree repository
         run: |
-          ostree commit --repo=repo --canonical-permissions --branch=screenshots/x86_64 flatpak_app/screenshots
+          ostree commit --repo=repo --canonical-permissions --branch=screenshots/${{ matrix.arch }} flatpak_app/screenshots
```
Comment (tytan652) [inline .github/workflows/flatpak.yml:166]: @GeorgesStavracas is this good ?

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -64,14 +64,43 @@ jobs:
...
+              echo 'create_artifacts=true' >> $GITHUB_OUTPUT
+            else
+              echo 'create_artifacts=false' >> $GITHUB_OUTPUT
+            fi
+
+            if test -n "$(curl -H "Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" -s "${{ github.event.pull_request.url }}" | jq -e '.labels[] | select(.name == "Flatpak aarch64")')"; then
+              echo 'flatpak_matrix=["x86_64", "aarch64"]' >> $GITHUB_OUTPUT
+            else
+              echo 'flatpak_matrix=["x86_64"]' >> $GITHUB_OUTPUT
+            fi
```
Comment (tytan652) [inline .github/workflows/main.yml:89]: Is `Flatpak aarch64` a good label name to enable aarch64 build on PRs ?

**Comment by tytan652** [comment]
NOTE for self: add an option to the upstream action to use flatpak-builder option `--stop-at=MODULENAME`

**Comment by tytan652** [comment]
Drafted waiting for PRs on flatpak action being reviewed and merged.

**Comment by Conan-Kudo** [comment]
> Disable SVT-AV1 on non-x86_64

SVT-AV1 builds on AArch64 just fine. We build it in Fedora for all architectures.

**Comment by tytan652** [comment]
> SVT-AV1 builds on AArch64 just fine. We build it in Fedora for all architectures.

When the PR was SVT-AV1 repo was explicitly stating that only x86_64 was supported, it has indeed changed since then.

Note, that PR is under a heavy rewrite.

**Comment by tytan652** [comment]
Closed over #9979

**Pull request closed without merging**